benchmark/src/main/scala/fs2/benchmark/PullBenchmark.scala

```diff
@@ -10,18 +10,20 @@ class PullBenchmark {
   var n: Int = _
 
   @Benchmark
-  def unconsPull(): Int =
-    (Stream
-      .chunk(Chunk.seq(0 to 2560)))
+  def unconsPull(): Int = {
+    val s: Stream[Pure, Int] = Stream
```

Comment:
Dotty seems to have trouble with inferring the type constructor through `repeatPull`. Without this change, Dotty complains:

```scala
[error] -- [E057] Type Mismatch Error: /Users/mpilquist/Development/oss/fs2/benchmark/src/main/scala/fs2/benchmark/PullBenchmark.scala:22:14
[error] 22 |      .covary[IO]
[error]    |              ^
[error]    |Type argument cats.effect.IO does not conform to lower bound [x] =>> Any
```